src/connector/src/sink/clickhouse.rs

```diff
@@ -351,9 +385,13 @@ impl ClickHouseSinkWriter {
                     data,
```

Comment:
After this PR, the behavior of `append_only` may be the same as `upsert`? If so we may keep only the `upsert`, and change the method name to a simple `write`.